completions/kldload

```diff
@@ -7,12 +7,16 @@ _kldload()
     local cur prev words cword
     _init_completion || return
 
-    local moddir=/modules/
-    [[ -d $moddir ]] || moddir=/boot/kernel/
+    local moddirs modules i
+    IFS=';' read -ra moddirs <<< "$(sysctl -n kern.module_path)"
```

Comment:
I know nothing about this, but could you elaborate on why use `sysctl -n kern.module_path` instead of `kldconfig -r`?

Also, should we fall back to the previous hardcoded dirs if the utility we choose to use here yields nothing? Should we output its stderr to `/dev/null` in any case?

Reply:
Will replace command and add error handling. Also I found `kldload(8)` should support files with absolute/relative paths, will add this.